tests/__snapshots__/test_export.ambr

```diff
@@ -49,16 +49,20 @@
    
   </style>
   <table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
-  <thead class="gt_header">  <tr>
-      <th colspan="4" class="gt_heading gt_title gt_font_normal">Data listing from <strong>exibble</strong>
+  <thead class="gt_header">
+    <tr>
+      <th colspan="4" class="gt_heading gt_title gt_font_normal">Data listing from <strong>exibble</strong></th>
```

Comment:
Here is the test that changed, based on this fix. There is a closing `</th>` element now.